movie/settings.py

```diff
@@ -0,0 +1,122 @@
+"""
+Django settings for movie project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+import os
+from pathlib import Path
+
+from dotenv import load_dotenv
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+load_dotenv()
+SECRET_KEY = os.getenv("SECRET_KEY")
```

Comment:
Ensure that the `SECRET_KEY` is set in the environment variables and not hardcoded in the source code. This is important for the security of your application.